cmd/gateway_server/main.go

```diff
@@ -31,8 +34,13 @@ func main() {
 		return
 	}
 
-	// Create a relay controller with a cached POKT client and the logger
-	relayController := controllers.NewRelayController(pokt_client_decorators.NewCachedClient(client), gatewayConfigProvider.GetAppStakes(), logger)
+	// Initialize a TTL cache for session caching
+	sessionCache := ttl_cache.NewTTLCacheClient[string, *models.GetSessionResponse]() // Initialize the cache client
+
+	sessionCache.Start() // Start the cache client
```

Comment:
sessionCache.start() needs to be in a goroutine, otherwise it will block the main process execution